maps/donut3.dmm

```diff
@@ -59245,21 +59475,15 @@
 /turf/simulated/floor/circuit/white,
 /area/station/turret_protected/AIsat)
 "rdH" = (
-/obj/machinery/power/data_terminal,
 /obj/cable{
-	d2 = 4;
-	icon_state = "0-4"
-	},
-/obj/machinery/light/emergency{
-	dir = 8;
-	pixel_x = -12
-	},
-/obj/machinery/sleeper/compact{
-	dir = 1
```

Comment:
you also appear to be making undocumented changes to the sleepers

Reply:
I must've accidentally deleted the sleeper and replaced it with one which is identical (see images).
![image](https://user-images.githubusercontent.com/87152856/129256361-3d2feb41-cced-4a18-8b4c-2fa826b1ac0a.png)
![image](https://user-images.githubusercontent.com/87152856/129256391-0f5562d0-8f16-4abb-8b75-60f22c22fe10.png)